welcome/about-kueue.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="about-kueue"]
+= About {product-title}
+:context: about-kueue
+
+toc::[]
+
+{product-title} is a Kubernetes-native system that manages quotas and how jobs consume them. Kueue decides when a job waits, when a job is admitted to start by creating pods, and when a job should be _preempted_ (active pods should be deleted).
+
+Kueue is compatible with environments that use heterogeneous, elastic resources. This means that the environment has many different resource types, and those resources are capable of dynamic scaling.
+
+{product-title} is based on the link:https://kueue.sigs.k8s.io/docs/[Kueue] open source project.
+
+// Personas
+[id="about-kueue-personas"]
+== {kueue-s} personas
+
+Different personas exist in a {kueue-s} workflow.
+
+Batch administrators:: Batch administrators manage the cluster infrastructure and establish quotas and queues.
+Batch users:: Batch users run workloads on the cluster. Examples of batch users might be researchers, AI/ML engineers, or data scientists.
+Serving users:: Serving users run workloads on the cluster. For example, to expose a trained AI/ML model for inference.
+Platform developers:: Platform developers integrate Kueue with other software. They might also contribute to the Kueue open source project.
+
+// How it works
+[id="about-kueue-workflow"]
+== {kueue-s} workflow
+// TODO: add diagram?
+
+The {kueue-s} workflow can be described at a high level as follows:
```

Comment:
This is very specific to the Job controller.

I'm not sure if you want this doc to only mean Kueue is used to manage batch jobs.

And you will expand on this in other sections?

Reply:
Updated step 5 to make this general